kinto_http/__init__.py

```diff
@@ -251,6 +251,14 @@ def get_bucket(self, bucket=None):
         try:
             resp, _ = self.session.request('get', endpoint)
         except KintoException as e:
+            error_resp_code = e.response.status_code
+            if error_resp_code == 401 or error_resp_code == 403:
+                msg = ("Unauthorized. Please check that the bucket exists and "
+                       "that you have the permission to create or write on "
+                       "this group.")
+                e = KintoException(msg, e)
```

Comment:
Why are we raising a `KintoException` there?
I wonder if we should rather raise a `KintoUnauthorizedAccess` error instead to make it easy for user to catch this specific error.